sessionctx/variable/sysvar.go

```diff
@@ -814,10 +822,20 @@ const (
 	BlockEncryptionMode = "block_encryption_mode"
 	// WaitTimeout is the name for 'wait_timeout' system variable.
 	WaitTimeout = "wait_timeout"
-	// ValidatePasswordNumberCount is the name of 'validate_password_number_count' system variable.
-	ValidatePasswordNumberCount = "validate_password_number_count"
+	// ValidatePasswordCheckUserName is the name of 'validate_password_check_user_name' system variable
+	ValidatePasswordCheckUserName = "validate_password_check_user_name"
+	// ValidatePasswordDictionaryFile is the name of 'validate_password_dictionary_file' system variable
+	ValidatePasswordDictionaryFile = "validate_password_dictionary_file"
 	// ValidatePasswordLength is the name of 'validate_password_length' system variable.
 	ValidatePasswordLength = "validate_password_length"
+	// ValidatePasswordMixedCaseCount is the name of 'validate_password_mixed_case_count' system variable
+	ValidatePasswordMixedCaseCount = "validate_password_mixed_case_count"
+	// ValidatePasswordNumberCount is the name of 'validate_password_number_count' system variable.
+	ValidatePasswordNumberCount = "validate_password_number_count"
+	// ValidatePasswordPolicy is the name of 'validate_password_policy' system variable
+	ValidatePasswordPolicy = "validate_password_policy"
+	// ValidatePasswordSpecialCharCount is the name of 'validate_password_special_char_count' system variable
+	ValidatePasswordSpecialCharCount = "validate_password_special_char_count"
```

Comment:
See https://dev.mysql.com/doc/refman/8.0/en/validate-password-options-variables.html#sysvar_validate_password.policy, it seems the names of variables are changed.

Reply:
If we use `validate_password.policy`, TiDB will meet an error. Seems that TiDB doesn't support `.` in sysvar

Reply:
Could you try to resolve that?

Reply:
TiDB parser recongizes`validate_password` as an identifier instead of `validate_password.policy`.   I'll try to mark it as  `NotKeywordToken` in parser to see if it works.

Reply:
@wjhuang2016 @SunRunAway These variable names are changed in MySQL 8.0. For now, TiDB is focused on being compatible with MySQL 5.7. We can change these names later in another PR.